src/main/scala/org/clulab/habitus/utils/ContextExtractor.scala

```diff
@@ -46,9 +48,13 @@ trait ContextExtractor {
       "harvesting"
     } else if (lemmas.exists(l => creditLemmas.contains(l))) {
       "credit"
+    } else if (lemmas.exists(l => irrigationLemmas.contains(l))) {
+      "irrigation"
+    } else if (lemmas.exists(l => weedsLemmas.contains(l))) {
+      "weeds"
```

Comment:
I think this is long enough to warrant something like this:
```scala
  case class ProcessToLemmas(process: String, lemmas: Set[String]) {
    def this(processAndLemmas: (String, Set[String])) = this(processAndLemmas._1, processAndLemmas._2)
  }

  // These are prioritized highest to lowest because there can be multiple matches.
  val processToLemmasMap = Seq(
    "planting"         -> Set("plant", "sow", "cultivate", "cultivation", "grow"),
    "harvest"          -> Set("harvest", "yield"),
    "credit"           -> Set("credit", "finance", "value"),
    "irrigation"       -> Set("irrigation", "irrigate"),
    "weeds"            -> Set("weed"),
    "natural_disaster" -> Set("flood", "bird", "attack")
  ).map(new ProcessToLemmas(_)) // just for convenience

  def getProcess(mention: Mention): String = {
    val lemmas = mention.sentenceObj.lemmas.get
    val process = processToLemmasMap
        .find { processToLemmas =>
          lemmas.exists(processToLemmas.lemmas)
        }
        .map(_.process)
        .getOrElse("UNK")

    process
  }
```